ckan/logic/schema.py

```diff
@@ -532,9 +534,9 @@ def default_follow_dataset_schema():
 
 def member_schema():
     schema = {
-        'id': [group_id_exists, unicode],
-        'username': [user_name_exists, unicode],
-        'role': [role_exists, unicode],
+        'id': [not_missing, not_empty, group_id_or_name_exists, unicode],
```

Comment:
not_empty is likely covered by group_id_or_name_exists (as below), but that's fine :smile: